pyTigerGraph/gds/dataloaders.py

```diff
@@ -415,10 +415,18 @@ def _request_kafka(
             resKey=None
         )
         # Check status
-        _stat_payload = {
-            "graph_name": tgraph.graphname,
-            "requestid": resp["request_id"],
-        }
+        try:
+            _stat_payload = {
+                "graph_name": tgraph.graphname,
+                "requestid": resp["request_id"],
+            }
+        except KeyError:
+            print(resp)
+            if resp["results"][0]["kafkaError"] != '':
+                raise TigerGraphException(
+                    "Error writing to Kafka: {}".format(res[0]["kafkaError"])
+                )
+
```

Comment:
Need to return here or it will go into the while loop below

Reply:
Fixed in next commit.